marko.json

```diff
@@ -183,5 +183,15 @@
             "@selected": "boolean"
         }
     },
-    "<ebay-select-option>": {}
+    "<ebay-select-option>": {},
+    "<ebay-textbox>": {
+        "renderer": "./src/components/ebay-textbox/index.js",
+        "@*": "expression",
+        "@html-attributes": "expression",
+        "@class": "string",
+        "@invalid": "boolean",
+        "@disabled": "boolean",
```

Comment:
Should we make disabled a pass through html attribute and remove it from here and in the component/template?

Reply:
👍 Let's do that for `disabled` here too. In other cases, we use `disabled` for internal behavior which is why it is an explicit attribute, but we don't have that yet for this component.